content/02.results.md

```diff
@@ -189,8 +221,77 @@ In XXX arabiensis individuals we identify 10,000,000 SNPs (Q% multiallelic), a S
 
 ## Population Structure
 
-## Genetic Diversity within Populations
+### Genome wide patterns
+
+%% TODO NOtes currently.
+
+Re-introduce key idea of structure being different across the genome. 
+
+How does arabiensis fit into this? Are there regions of the genome where arabiensis ancestry is secondary?
+
+
+### PCA / UMAP
+
+To highlight population structure we performed principal component analysis across all wild-caught samples in the dataset.
+
+To avoid confounding of structure in genomic regions including paracentric inversions, extremely low diversity and regions under strong selection, we limited our analysis to euchromatic regions of chromosome 3L. 
+
+The most apparent signal in the dataset is PC1 clearly being driven by Arabiensis, with clear separation of arabiensis samples from gambiae/coluzzii. 
+
+The apparent hybrid sits between gambiae and arabiensis samples.
+
+To view population structure within gambiae/coluzzii and arabiensis more independently, we performed subsequent PCA analysis Arabiensis and gambiae/coluzzii individuals separately. 
+
+Population structure between gambiae and coluzzii is significantly more complex. 
+
+
+
+Separately between species. What are the major findings?
+
+- Arabiensis drives PC1. 
+
+- East Africa: Seems to be clear population structure between _gambiae_ in KE and TZ.
+
+- According to AIM analysis, a significant proportion of samples in these groups are classed as IM between gambiae. Certainly not coluzzii, but some kind of complex ancestry. 
+
+- Relevance to TENEGLRA
+
+- West Africa- in far west Africa we see intermediate population. Not gambiae coluzzii, unlikely to be hybrids, but a related subspecies. 
+
+Interestingly seems to be stable in the presence of both gamb and colu. Although they sit close to col in the PCA they are distinct from coluzzii, given they are founf at the same site.
```

Comment:
I didn't realise there were normal gamb or colu from the same sites as the 'hybrid/cryptic' samples in the far west.